Makefile

```diff
@@ -31,12 +32,18 @@ client:
 	go build ${LDFLAGS} -o ${BINARY}-client . ; \
 	cd - >/dev/null
 
+vet:
+	-cd ${BUILD_DIR}; \
```

Comment:
Since we are in the project, why we need to `cd` again ?
We usually call the `Makefile ...`  in the same directory
And the `-cd` is typo ?